### PR TITLE
Fix/update to new class registry

### PR DESCRIPTION
**Pull request by qejk**

### CODE REVIEW

**Comment by rhyslbw** [comment]
space:testing is now @ 2.0.1

File: package.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ Package.onUse(function(api) {
   api.use([
     'check',
     'ecmascript',
-    'space:messaging@2.0.0',
+    'space:messaging@2.1.0',
```
Comment (rhyslbw) [inline package.js]: We shouldn't need this now space.domain is providing ValueObject

File: source/quantity.js [hunk 2]
```diff
@@ -53,17 +53,16 @@ Quantity = Space.domain.ValueObject.extend('Quantity', {
...
+  _getCleanValue(other) {
     return (other instanceof Quantity) ? other.value : other;
   }
 
 });
 
-// Register EJSON type
 Quantity.type('Quantity');
 
 Quantity.ERRORS = {
```
Comment (darko-mijic) [inline source/quantity.js:66]: I would move this inside the class

File: source/quantity.js [hunk 3]
```diff
@@ -53,17 +53,16 @@ Quantity = Space.domain.ValueObject.extend('Quantity', {
...
 
-  _getCleanValue: function(other) {
+  _getCleanValue(other) {
     return (other instanceof Quantity) ? other.value : other;
   }
 
 });
 
-// Register EJSON type
 Quantity.type('Quantity');
```
Comment (darko-mijic) [inline source/quantity.js]: Type registration is done by Space.messaging.Serializable so this should not be needed

**Pull request merged**